sample-app/src/components/DirectAnswer.tsx

```diff
@@ -0,0 +1,57 @@
+import { useAnswersState } from "@yext/answers-headless-react";
+import { FeaturedSnippetDirectAnswer, FieldValueDirectAnswer } from '@yext/answers-headless';
+import renderWithHighlighting from './utils/renderWithHighlighting';
+import classNames from "classnames";
+import '../sass/DirectAnswer.scss';
+
+interface DirectAnswerProps {
+  cssClasses?: {
+    container?: string,
+    title?: string,
+    content?: string,
+    description?: string
+  }
+}
+
+const defaultCSSClasses = {
+  container: 'DirectAnswer',
+  title: 'DirectAnswer__title',
+  content: 'DirectAnswer__content',
+  description: 'DirectAnswer__contentDescription'
+};
+
+function isFeaturedSnippetDirectAnswer(
+  answer: FeaturedSnippetDirectAnswer | FieldValueDirectAnswer
+): answer is FeaturedSnippetDirectAnswer {
+  return 'snippet' in answer;
+}
+
+export default function DirectAnswer(props: DirectAnswerProps): JSX.Element | null {
+  const directAnswerResult = useAnswersState(state => state.directAnswer.result);
+  const isLoading = useAnswersState(state => state.searchStatus.isLoading);
+  if (!directAnswerResult) {
+    return null;
+  }
+  const { cssClasses:customCssClasses } = props;
+  const cssClasses = Object.assign(defaultCSSClasses, customCssClasses);
+  const containerCssClasses = classNames(cssClasses.container, { [`${cssClasses.container}--loading`]: isLoading });
```

Comment:
Can we add an extra cssClass prop for `containerLoading`? If we want to allow tailwind customization, the `cssClasses.container` value would contain an array of tailwind classes, and adding `--loading` to the end of that wouldn't work

Reply:
I see, will do.